Framework/API/src/PeakFunctionIntegrator.cpp

```diff
@@ -8,7 +8,9 @@
 
 #include "MantidAPI/FunctionDomain1D.h"
 #include "gsl/gsl_errno.h"
+#include <boost/math/distributions/students_t.hpp>
```

Comment:
Where is this used?

Reply:
good catch. This is a leftover from a previously tried implementation of the error. I'll delete it.

Reply:
I just deleted the unused headers.
@KedoKudo Can you please approve again after these minor change?